scripts/nuget_patch/generate_nuget_spec.py

```diff
@@ -0,0 +1,59 @@
+import os
+import shutil
+import sys
+from pathlib import Path
+
+WORK_DIR = Path(".")
+DOTNET_DLL_DIR = WORK_DIR / sys.argv[1]
+WINDOWS_RUNTIME_DIR = WORK_DIR / sys.argv[2]
+LINUX_RUNTIME_DIR = WORK_DIR / sys.argv[3]
+VERSION = sys.argv[4][1:]
+
+SPEC_FILE = WORK_DIR / "Package.nuspec"
+LICENSE_FILE = WORK_DIR / "LICENSE"
+shutil.copy(LICENSE_FILE, str(LICENSE_FILE) + ".txt" )
+LICENSE_FILE = Path( str(LICENSE_FILE )+ ".txt" )
+
+COPYRIGHT_FILE = WORK_DIR / "scripts" / "copyright_header.txt"
+COPYRIGHT = open(COPYRIGHT_FILE,'r').read()[3:]
+
+f = open(SPEC_FILE, 'w')
+f.write('<?xml version="1.0" encoding="utf-8"?>\n')
+f.write('<package>\n')
+f.write('\t<metadata>\n')
+f.write('\t\t<id>MeshLib</id>\n')
+f.write('\t\t<version>' + VERSION + '</version>\n')
```

Comment:
Might be better to create a template file and apply changes with [string.Template](https://docs.python.org/3/library/string.html#string.Template)
There is a usage example in scripts/wheel/build_wheel.py